[2024-10-12T00:16:29.472-0500] {processor.py:161} INFO - Started process (PID=86) to work on /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:16:29.485-0500] {processor.py:830} INFO - Processing file /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py for tasks to queue
[2024-10-12T00:16:29.541-0500] {logging_mixin.py:188} INFO - [2024-10-12T00:16:29.533-0500] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:16:29.808-0500] {logging_mixin.py:188} INFO - [2024-10-12T00:16:29.728-0500] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py", line 9, in <module>
    from common.add_task import task_monitoringmodel_v2
  File "/opt/airflow/dags/common/add_task.py", line 4, in <module>
    import joblib
ModuleNotFoundError: No module named 'joblib'
[2024-10-12T00:16:29.824-0500] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:16:30.977-0500] {processor.py:183} INFO - Processing /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py took 2.733 seconds
[2024-10-12T00:17:04.461-0500] {processor.py:161} INFO - Started process (PID=105) to work on /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:17:04.462-0500] {processor.py:830} INFO - Processing file /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py for tasks to queue
[2024-10-12T00:17:04.511-0500] {logging_mixin.py:188} INFO - [2024-10-12T00:17:04.508-0500] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:17:04.866-0500] {logging_mixin.py:188} INFO - [2024-10-12T00:17:04.839-0500] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py", line 9, in <module>
    from common.add_task import task_monitoringmodel_v2
  File "/opt/airflow/dags/common/add_task.py", line 4, in <module>
    import joblib
ModuleNotFoundError: No module named 'joblib'
[2024-10-12T00:17:04.873-0500] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:17:05.298-0500] {processor.py:183} INFO - Processing /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py took 0.861 seconds
[2024-10-12T00:17:35.378-0500] {processor.py:161} INFO - Started process (PID=123) to work on /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:17:35.379-0500] {processor.py:830} INFO - Processing file /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py for tasks to queue
[2024-10-12T00:17:35.381-0500] {logging_mixin.py:188} INFO - [2024-10-12T00:17:35.381-0500] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:17:35.412-0500] {logging_mixin.py:188} INFO - [2024-10-12T00:17:35.405-0500] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py", line 9, in <module>
    from common.add_task import task_monitoringmodel_v2
  File "/opt/airflow/dags/common/add_task.py", line 4, in <module>
    import joblib
ModuleNotFoundError: No module named 'joblib'
[2024-10-12T00:17:35.414-0500] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:17:35.632-0500] {processor.py:183} INFO - Processing /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py took 0.261 seconds
[2024-10-12T00:18:06.230-0500] {processor.py:161} INFO - Started process (PID=142) to work on /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:18:06.232-0500] {processor.py:830} INFO - Processing file /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py for tasks to queue
[2024-10-12T00:18:06.235-0500] {logging_mixin.py:188} INFO - [2024-10-12T00:18:06.234-0500] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:18:06.268-0500] {logging_mixin.py:188} INFO - [2024-10-12T00:18:06.261-0500] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py", line 9, in <module>
    from common.add_task import task_monitoringmodel_v2
  File "/opt/airflow/dags/common/add_task.py", line 4, in <module>
    import joblib
ModuleNotFoundError: No module named 'joblib'
[2024-10-12T00:18:06.270-0500] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py
[2024-10-12T00:18:06.292-0500] {processor.py:183} INFO - Processing /opt/airflow/dags/04_four_dag/monitor_maintain_dag_v1-4.py took 0.068 seconds
